src/Generate.purs

```diff
@@ -146,44 +146,39 @@ INPUTS
 in {
   inherit inputs;
 
-  installSpagoStyle = pkgs.runCommand "install-spago-style" {} ''
-      >>$out echo "#!/usr/bin/env bash"
-      >>$out echo
-      >>$out echo "echo installing dependencies..."
-      >>$out echo "${builtins.toString (
-        builtins.map cpPackage (builtins.attrValues inputs))}"
-      >>$out echo "echo done."
-      chmod +x $out
+  installSpagoStyle = pkgs.writeShellScriptBin "install-spago-style" ''
+    echo "installing dependencies..."
+    ${builtins.toString (
+       builtins.map cpPackage (builtins.attrValues inputs))}
+    echo "done."
   '';
 
-  buildSpagoStyle = pkgs.runCommand "build-spago-style" {} ''
-      >>$out echo "#!/usr/bin/env bash"
-      >>$out echo
-      >>$out echo "echo building project..."
-      >>$out echo "purs compile ${builtins.toString (
-        builtins.map getGlob (builtins.attrValues inputs))}" \"\$@\"
-      >>$out echo "echo done."
-      chmod +x $out
+  buildSpagoStyle = pkgs.writeShellScriptBin "build-spago-style" ''
+    EXTRA_FILES=$@
+
+    echo "echo building project..."
+    echo "purs compile ${builtins.toString (
```

Comment:
Actually this should also depend on `purs`/`purescript`. Currently it circumvents nix. But I did not change it as maybe the whole function is not needed anymore now that we have `mkBuildProjectOutput`.
Otherwise it could be `${purescript}/bin/purs`.

Reply:
i actually do use this script quite often, since mkBuildProjectOutput is the opposite of what i want to use when developing. but yes, changing it to using purescript from easy-purescript-nix is probably fine